models/migrations/migrations.go

```diff
@@ -497,6 +498,11 @@ var migrations = []Migration{
 	NewMigration("Add PinOrder Column", v1_20.AddPinOrderToIssue),
 	// v259 -> 260
 	NewMigration("Convert scoped access tokens", v1_20.ConvertScopedAccessTokens),
+
+	// Gitea 1.21.0 ends at 260
```

Comment:
Wait a minute, why do you change it to `1.21.0` in 6cc62e9b6d5c880606813ab90d8db720bac7a6b7 ? Please be more careful.

Reply:
Alright, I see #25243.

Reply:
will be more careful next time...